internal/oci/runtime_oci.go

```diff
@@ -995,6 +996,9 @@ func (r *runtimeOCI) DeleteContainer(ctx context.Context, c *Container) error {
 	}
 
 	_, err := r.runtimeCmd("delete", "--force", c.ID())
+	if errors.Is(err, ErrNotFound) {
+		return nil
```

Comment:
Should we lock something here? Which use case are we skipping?

Reply:
I don't remember the exact situation but we hit this in a couple of integration tests. I don't think it was ultimately fatal but I think it's good to have